src/components/Image/index.js

```diff
@@ -69,4 +69,6 @@ const ImageWithOnyx = React.memo(
     imagePropsAreEqual,
 );
 ImageWithOnyx.resizeMode = RESIZE_MODES;
+
+export {RESIZE_MODES};
```

Comment:
@pasyukevich Is there any reason we are exporting `RESIZE_MODES` from here again? why can't we use it from `./resizeModes`?

Reply:
Yes, to keep approach that used before and have all that is needed for the component from one file

Reply:
@pasyukevich Files should be named after the component/function/constants they export, respecting the casing used for it. read more [here](https://github.com/Expensify/App#file-namingstructure)

if there is no strong reason can we just use values from `./resizeModes` everywhere? this will make it less confusing.

Reply:
Updated